_pages/plugins/motiq.md

```diff
@@ -0,0 +1,27 @@
+---
+mediawiki: MotiQ
+title: MotiQ
+description: A set of ImageJ plugins to quantify dynamics, morphology, and fluorescence of microglial (or other) cells.
+categories: [Automation, Analysis, Particle Analysis, Segmentation, Object Detection, Neuron, Tracking, Skeleton]
+doi: 10.1091/mbc.E21-11-0585
+source-url: https://github.com/hansenjn/MotiQ
+dev-status: Stable
+support-status: Active
+team-founder: '@hansenjn'
+team-maintainer: '@hansenjn'
+---
+
+MotiQ is s set of ImageJ plugins to quantify dynamics, morphology, and fluorescence of microglial (or other) cells.
+
+The software is published with the following publication:
+Jan N. Hansen, Matthias Brückner, Marie J. Pietrowski, Jan F. Jikeli, Monika Plescher, Hannes Beckert, Mareike Schnaars, Lorenz Fülle, 
+Katharina Reitmeier, Thomas Langmann, Irmgard Förster, Delphine Boche, Gabor C. Petzold, and Annett Halle. 
+*MotiQ: an open-source toolbox to quantify the cell motility and morphology of microglia.*
+Molecular biology of the cell, 33:11 (2022). doi: https://doi.org/10.1091/mbc.E21-11-0585
```

Comment:
Please use the `{% include citatation}` mechanism to cite publications. See https://imagej.net/editing/citations